<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login_page">
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-image: url(:/src/background.jpg);
}	
QLineEdit{
color:#8d98a1;
background-color:#405361;
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton{
background:#ced1d8;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color:rgb(224,0,0);
border-style:inset;
font-style:MingLiU-ExtB;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}</string>
     </property>
     <widget class="QPushButton" name="login_btn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>380</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_line">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>290</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_line">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>230</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="information_lab">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>347</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>430</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>By:德灵风雨 沙拉肉段 Lutio0125
V0.1Beta</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainWindow_page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>801</width>
        <height>501</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>班级</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>事件</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>时间</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>详情</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="export_btn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>560</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>导出Excel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="return_btn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>560</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="lineNumber_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>共有n条信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delInformation_btn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>560</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除选中</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>441</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>搜索条件：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="serchItem">
         <item>
          <property name="text">
           <string>ID</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>姓名</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>时间</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>班级</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="serchInformation_line">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>搜索内容...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="serch_btn">
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="mainWindow_page_2">
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-image: url(:/src/img1.png);
}
QLineEdit{
color:#8d98a1;
background-color:#405361;
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton{
background:#ced1d8;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color:rgb(224,0,0);
border-style:inset;
font-style:MingLiU-ExtB;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>360</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toSerchWindow_btn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>280</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>查询/删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insert_btn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>200</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>录入/修改</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="insert_page">
     <widget class="QPushButton" name="return_btn_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>560</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>771</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>录入</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>姓名*：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="studentName_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>班级*：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="studentClass_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>日期：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QCalendarWidget" name="studentDate_date">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>30</y>
         <width>341</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>事件*：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="studentEvent_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>详细事件：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>270</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>录入</string>
       </property>
      </widget>
      <widget class="QLabel" name="tip_lab">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>270</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="stuedntEventSpecific_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>311</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>771</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>修改</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>事件编号：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="ID_line">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="change_Com">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>姓名</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>班级</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>事件</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>详细事件</string>
        </property>
       </item>
      </widget>
      <widget class="QTextEdit" name="changeValue_line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>411</width>
         <height>121</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>更改内容....</string>
       </property>
      </widget>
      <widget class="QPushButton" name="change_btn">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>90</y>
         <width>321</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
      <widget class="QLabel" name="changeTip_lab">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>30</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
